packages/backend/src/rapyd/wallet/request.ts

```diff
@@ -37,6 +38,84 @@ const rapydVerifyIdentity = async (req: RapydIdentityRequest) => {
   return await makeRapydPostRequest('identities', JSON.stringify(req))
 }
 
+const rapydDepositLiquidity = async (req: RapydDepositRequest) => {
+  return await makeRapydPostRequest('account/deposit', JSON.stringify(req))
+}
+
+const rapydHoldLiquidity = async (req: RapydHoldRequest) => {
+  return await makeRapydPostRequest('account/balance/hold', JSON.stringify(req))
+}
+
+const rapydReleaseLiquidity = async (req: RapydReleaseRequest) => {
+  return await makeRapydPostRequest(
+    'account/balance/release',
+    JSON.stringify(req)
+  )
+}
+
+const rapydTransferLiquidity = async (
+  req: RapydTransferRequest,
+  withAccept?: boolean
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+): Promise<any> => {
+  if (!withAccept) {
+    return await makeRapydPostRequest('account/transfer', JSON.stringify(req))
+  }
+  const transferResponse = await makeRapydPostRequest(
+    'account/transfer',
+    JSON.stringify(req)
+  )
+  if (transferResponse.status.status !== 'SUCCESS') {
+    if (
+      transferResponse.status.error_code === 'NOT_ENOUGH_FUNDS' &&
+      req.source_ewallet === env.RAPYD_SETTLEMENT_EWALLET
+    ) {
+      // await handleSettlementOutOfFunds(req, env.RAPYD_SETTLEMENT_EWALLET)
```

Comment:
I wanted to already include this case:

Whenever an incoming payment completes, and a receiver should get the money from the settlement account, in case the settlement account is out of funds, Fund it and then retry the (settlement -> receiver) transfer.

However, the eWallet that we are using as a settlement is of type Collect, not of type Personal. As such, we cannot simply fund it. If I use the rapyd endpoint that we use to add money to personal wallets, the response is an equivalent to a not found error, even though the ewallet clearly exists, but as a Collect type.

The only way I managed to add money into it was by creating a bank payment from the rapyd dashboard, and then doing a fake bank transfer to it. I then had to also change the payment's state to completed manually and then collect the money within it.
So a lot of work in order to fund the settlement account.

I'm dropping this feature for now, as the edge case shouldn't really happen.